BUILD.ndk_sysroot.tpl

```diff
@@ -24,7 +24,7 @@ filegroup(
     srcs = glob([
         # "usr/lib/%s/**/*.so" % target_system_name,
         # "usr/lib/%s/**/*.a" % target_system_name,
-    ]),
+    ], allow_empty = True),
```

Comment:
The glob patterns here are already commented out, so this glob() will already return nothing FYI.

Reply:
Yes, I did not know much how to proceed here. If one day this comment is removed then allowed empty should also be removed